pkg/BUILD.bazel

```diff
@@ -478,6 +478,16 @@ ALL_TESTS = [
 # [1] https://docs.bazel.build/versions/master/be/general.html#test_suite
 # [2] https://docs.bazel.build/versions/master/be/common-definitions.html#common-attributes-tests
 
+test_suite(
+    name = "all_tests",
+    tags = [
+        "-broken_in_bazel",
+        "-flaky",
```

Comment:
Do we care about excluding flakey and integration tests here?

Reply:
I still included flaky tests but am leaving out `integration` for now. Generally these are the tests guarded by build tags (`compose`, `acceptance`, `lint`) that don't really make sense in this context anyway.